src/services/ContactRepository.ts

```diff
@@ -133,7 +133,7 @@ class ContactRepository {
         PersistentStore.store.getState()
       )
       const contact = {
-        did: { address: identifier },
+        did: { identifier: identifier },
```

Comment:
Change request: 
Change syntax to "identifier" directly

Reply:
Done.